sos/policies/distros/debian.py

```diff
@@ -50,12 +50,31 @@ def check(cls, remote=''):
 
     def dist_version(self):
         try:
-            with open('/etc/lsb-release', 'r') as fp:
-                rel_string = fp.read()
-                if "wheezy/sid" in rel_string:
+            with open('/etc/os-release', 'r') as fp:
+                rel_string = ""
+                lines = fp.readlines()
+                for line in lines:
+                    if "VERSION_CODENAME" in line:
+                        rel_string = line.split("=")[1].strip()
+                        break
+                if "squeeze" in rel_string:
                     return 6
-                elif "jessie/sid" in rel_string:
+                elif "wheezy" in rel_string:
                     return 7
+                elif "jessie" in rel_string:
+                    return 8
+                elif "stretch" in rel_string:
+                    return 9
+                elif "buster" in rel_string:
+                    return 10
+                elif "bullseye" in rel_string:
+                    return 11
+                elif "bookworm" in rel_string:
+                    return 12
+                elif "trixie" in rel_string:
+                    return 13
+                elif "forky" in rel_string:
+                    return 14
```

Comment:
I think a more readable version of this would be to use a dict to map release strings to version numbers, and then have a single conditional check like

```python
if rel_string in rel_versions:
    return rel_versions[rel_string]
```